include/pocl.h

```diff
@@ -483,10 +483,16 @@ typedef union
   _cl_command_svm_memadvise mem_advise;
 } _cl_command_t;
 
+typedef enum
+{
+  COMMAND_FAILED = -1,
```

Comment:
```suggestion
  POCL_COMMAND_FAILED = -1,
```
And if you don't need to force the enum number, better just leave it out. 0 might have significance as it's the default if zero initialized.